CHANGELOG.next.toml

```diff
@@ -11,6 +11,22 @@
 # meta = { "breaking" = false, "tada" = false, "bug" = false, "target" = "client | server | all"}
 # author = "rcoh"
 
+[[smithy-rs]]
+message = """
+Replace all usages of `AtomicU64` with `AtomicUsize` to support 32bit targets.
+"""
+references = ["smithy-rs#1811"]
+meta = { "breaking" = false, "tada" = false, "bug" = false, "target" = "server"}
+author = "LukeMathWalker"
+
+[[smithy-rs]]
+message = """
+Replace all usages of `AtomicU64` with `AtomicUsize` to support 32bit targets.
+"""
+references = ["smithy-rs#1811"]
+meta = { "breaking" = true, "tada" = false, "bug" = false, "target" = "client"}
+author = "LukeMathWalker"
```

Comment:
It's breaking only for the client - let me know if there are better ways to convey it.